### PR TITLE
Feat/2830 (backend) Support include_workspace_ids in the known_members endpoint

### PR DESCRIPTION
**Pull request by raphj**
#2830 

Some random cleanups and fixes related to types snicked in this MR.

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works [not relevant]
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done [not relevant]

### CODE REVIEW

File: backend/tracim_backend/exceptions.py [hunk 1]
```diff
@@ -463,6 +463,10 @@ class TooShortAutocompleteString(TracimException):
     pass
 
 
+class CannotUseBothIncludeAndExcludeWorkspaceUsers(TracimException):
```
Comment (inkhey) [inline backend/tracim_backend/exceptions.py:466]: If this error occured in endpoint , we get a error 500, right ?
I suggest you to either does a marshmallow check like this one https://github.com/tracim/tracim/blob/3fde6f6abaf7401f890a11d05bd680d5395ee717/backend/tracim_backend/views/core_api/schemas.py#L394
or add : 
- specific error code
- catching of this error as 400 in correct endpoint

Comment (raphj) [reply backend/tracim_backend/exceptions.py:466]: done, using:

```
@hapic.handle_exception(CannotUseBothIncludeAndExcludeWorkspaceUsers, HTTPStatus.BAD_REQUEST)
```

File: backend/tracim_backend/views/core_api/schemas.py [hunk 2]
```diff
@@ -593,25 +593,32 @@ def make_path_object(self, data: typing.Dict[str, typing.Any]) -> object:
...
         example="1,5",
-        description="comma separated list of excluded user",
+        description="comma separated list of excluded users",
     )
+
     exclude_workspace_ids = StrippedString(
         validate=regex_string_as_list_of_int,
         example="3,4",
-        description="comma separated list of excluded workspace: user of this workspace are excluded from result",
+        description="comma separated list of excluded workspaces: members of this workspace are excluded from the result",
```
Comment (inkhey) [inline backend/tracim_backend/views/core_api/schemas.py:610]: Maybe add note somewhere about the fact that exclude_workspace_ids and include_workspace_ids are mutually exclusive. Theses description are use in swagger autogenerated documentation.

File: backend/tracim_backend/exceptions.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/exceptions.py:466]: nice.

File: backend/tracim_backend/views/core_api/schemas.py (hunk 2 above)
Comment (raphj) [reply backend/tracim_backend/views/core_api/schemas.py:610]: ah, right

Comment (raphj) [reply backend/tracim_backend/views/core_api/schemas.py]: done

**Comment by inkhey** [comment]
@raphj  Is this ok to be merged ?

**Comment by raphj** [comment]
Yes :-)

**Pull request merged**